src/lib/drivers/driver.js

```diff
@@ -86,7 +90,7 @@ class DriverBase {
     return Promise.reject(new Error('Not implemented'));
   }
 
-  gotoURL(url) {
+  gotoURL(url, waitForLoad) {
```

Comment:
http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html

let's pass a hash of options instead :)